src/main/pages/technology/Technology.js

```diff
@@ -53,21 +53,27 @@ const Technology = () => {
               title: "Architecture",
               icon: TechnologyIcons.ArchitectureIcon,
               content: (
-                <ExpandableImage expandable image={ArchitectureDiagram} imageClassName="component-test__image" />
+                <div className="tabbed-info-box--shadow">
```

Comment:
I think the proper class for this is .tabbed-info-box__content--shadow, or use a more general .shadow class

.tabbed-info-box--shadow implies that this is a variant of tabbed-info-box, but it's tabbed-info-box__content.